torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -704,16 +705,18 @@ class LinearModelEncoder(StypeEncoder):
     features in a batched manner.
 
     Args:
-        in_channels (int): The dimensionality of the embedding feature.
-            Needs to be specified manually.
-        model (torch.nn.Module): Input to the model is a dictionary of
-            :obj:`MultiNestedTensor` and model outputs three-dimensional
-            embedding features that will be input into the linear layer.
+        col_to_model_cfg (dict): A dictionary mapping column names to
+            :class:`~torch_frame.config.ModelConfig`, which specifies a model
+            to map per-column :class:`TensorData` object of shape
+            :obj:`[batch_size, 1, *]` into row embeddings of shape
+            :obj:`[batch_size, 1, model_out_channels]`.
     """
 
     # NOTE: We currently support text embeddings but in principle, this encoder
     # can support any model outputs embeddings, including image/audio/graph
     # embeddings.
+    # TODO: Support more, stype.numerical, stype.multinested. We can support
+    # MLP model as well.
```

Comment:
I think this class can support so many stypes, including very basic ones like numerical. I think it is nice to add MLP example on later.